requirements/install.txt

```diff
@@ -3,3 +3,4 @@ numpy < 1.17 ; python_version <= '2.7'
 numpy ; python_version >= '3.0' and python_version < '3.12'
 # pillow < 3.0
 six
+typing-extensions; python_version < '3.10'
```

Comment:
It looks like perhaps a space is needed before the semicolon?

Reply:
Syntactically not needed, but I'm more than happy to change it to match the style used in the rest of the file.

Currently working though issued shown to me through the buildbot, so I'll get to it after I solve all of those issues that come up.

BTW, I was looking for a CONTRIBUTING.md or a style guide somewhere but didn't find anything, am I missing it?